node/mock-test.ts

```diff
@@ -163,10 +190,18 @@ export class MockTestRunner {
             return pathToExe;
         }
         else {
-            return this.downloadNode(downloadVersion, downloadDestination);
+            const result = await this.downloadNode(downloadVersion, downloadDestination);
+            return result;
         }
     }
 
+    private getNodePathSync(nodeVersion?: number): string {
```

Comment:
are we need this to be sync if it's private?

Reply:
it is a part of public sync method. In terms of to make it backward compatible I should keep it here